### PR TITLE
Docs: minor changes

### PR DESCRIPTION
**Pull request by aviv926**
This PR adds minor changes to the documentation but are necessary to keep it up to date.

It includes several different parts of the documentation but in order not to open many separate PR(s) I decided to consolidate the small changes into one PR.

### CODE REVIEW

File: docs/docs/administration/oauth.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Unable to set `app.immich:/` as a valid redirect URI? See [Mobile Redirect URI](
 
 ## Overview
 
-Immich supports 3rd party authentication via [OpenID Connect][oidc] (OIDC), an identity layer built on top of OAuth2. OIDC is supported by most identity providers, including:
+Immich supports 3rd party authentication via [OpenID Connect][oidc]https://openid.net/connect/ (OIDC), an identity layer built on top of OAuth2. OIDC is supported by most identity providers, including:
```
Comment (bo0tzz) [inline docs/docs/administration/oauth.md:11]: This seems wrong, and the link already worked fine afaict.

File: docs/docs/features/libraries.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ In all above scan methods, Immich will check if any files are missing. This can
 
 Finally, files can be deleted from Immich via the `Remove Offline Files` job. This job can be found by the three dots menu for the associated external storage that was configured under Administration > Libraries (the same location described at [create external libraries](#create-external-libraries)). When this job is run, any assets marked as offline will then be removed from Immich. Run this job whenever files have been deleted from the file system and you want to remove them from Immich.
 
+In order to remove offline files that linked to spesifce paths, remove paths -> Scan for new files -> Then run remove offline files
```
Comment (bo0tzz) [inline docs/docs/features/libraries.md:43]: What is this trying to say?

File: docs/docs/guides/remote-machine-learning.md [hunk 3]
```diff
@@ -18,6 +22,8 @@ services:
     container_name: immich_machine_learning
     # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
     # Example tag: ${IMMICH_VERSION:-release}-cuda
+    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
+    # Example tag: ${IMMICH_VERSION:-release}-cuda
```
Comment (bo0tzz) [inline docs/docs/guides/remote-machine-learning.md:26]: Duplicate?

File: docs/docs/install/docker-compose.mdx [hunk 4]
```diff
@@ -7,7 +7,8 @@ import ExampleEnv from '!!raw-loader!../../../docker/example.env';
 
 # Docker Compose [Recommended]
 
-Docker Compose is the recommended method to run Immich in production. Below are the steps to deploy Immich with Docker Compose.
+Docker Compose is the recommended method to run Immich in production. Below are the steps to deploy Immich with Docker Compose.  
+Supported Docker Compose 2.x. version.
```
Comment (bo0tzz) [inline docs/docs/install/docker-compose.mdx:11]:
```suggestion
Docker Compose is the recommended method to run Immich in production. Below are the steps to deploy Immich with Docker Compose.  
Immich requires Docker Compose version 2.x.
```

File: docs/docs/partials/_user-create.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
-If you have friends or family members who want to use the application as well, you can create addition accounts. The default password is `password`, and the user can change their password after logging in to the application for the first time.
+If you have friends or family members who want to use the application as well, you can create addition accounts. The default password is `password`, and the user have to change their password after logging in to the application for the first time, the system administrator can disable this option by unchecking the option "Require user to change password on first login" inside user registration interface.
```
Comment (bo0tzz) [inline docs/docs/partials/_user-create.md:1]:
```suggestion
If you have friends or family members who want to use the application as well, you can create addition accounts. The default password is `password`, and the user has to change their password after logging in to the application for the first time. The system administrator can disable this option by unchecking the option "Require user to change password on first login" in the user registration interface.
```

File: docs/src/pages/milestones.tsx [hunk 6]
```diff
@@ -51,12 +51,24 @@ import {
...
 } from '@mdi/js';
 import Layout from '@theme/Layout';
 import React from 'react';
 import Timeline, { DateType, Item } from '../components/timeline';
 
 const items: Item[] = [
+  {
+    icon: mdiMagnifyScan,
+    description:
+      'Advanced search with filters by date, location and more, also single search returns up to 5000 results compared to 100 in previous versions',
```
Comment (bo0tzz) [inline docs/src/pages/milestones.tsx:65]:
```suggestion
      'Advanced search with filters by date, location and more',
```

File: docs/docs/FAQ.mdx [hunk 7]
```diff
@@ -40,6 +40,11 @@ If you still cannot login to the app, try the following:
 
 ## Assets
 
+### Does Immich change the file?
+
+No, Immich does not touch the original file under any circumstances,  
+all changes of the file are saved in the Side Car file and in the database.
```
Comment (benmccann) [inline docs/docs/FAQ.mdx:46]: Is the side car file used only for photos that have an existing side car or will Immich create one if it doesn't exist?
```suggestion
all changes of the file are saved in the side car file and in the database.
```

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Thank you! Just some nitpicks

File: docs/docs/FAQ.mdx [hunk 8]
```diff
@@ -82,6 +87,11 @@ There are three thubmanil jobs for each asset:
 
 Also, there are additional jobs for person (face) thumbnails.
 
+### Files from WhatsApp do not appear with a correct date, why?
+
+Files sent on WhatsApp are saved without metadata on the file,
+therefore Immich has no way of knowing what the original date of the file is when files are uploaded from WhatsApp not in the order of arrival on the device.
```
Comment (benmccann) [inline docs/docs/FAQ.mdx:93]:
```suggestion
therefore Immich has no way of knowing what the original date of the file is when files are uploaded from WhatsApp not in the order of arrival on the device. [See #3527](https://github.com/immich-app/immich/issues/3527).
```

File: docs/docs/features/libraries.md (hunk 2 above)
Comment (benmccann) [inline docs/docs/features/libraries.md:43]:
```suggestion
In order to remove offline files that linked to specific paths, remove paths -> Scan for new files -> Then run remove offline files
```

File: docs/src/pages/milestones.tsx (hunk 6 above)
Comment (benmccann) [inline docs/src/pages/milestones.tsx:65]:
```suggestion
      'Advanced search with filters by date, location and more. Single search returns up to 5000 results compared to 100 in previous versions',
```

File: docs/docs/features/libraries.md (hunk 2 above)
Comment (aviv926) [reply docs/docs/features/libraries.md]: > What is this trying to say?

the correct order for deletion
https://discord.com/channels/979116623879368755/994044917355663450/1204154596067385354

File: docs/docs/administration/oauth.md (hunk 1 above)
Comment (aviv926) [reply docs/docs/administration/oauth.md:11]: Yes, I was sure I removed it. fixed

File: docs/docs/FAQ.mdx (hunk 7 above)
Comment (aviv926) [reply docs/docs/FAQ.mdx:46]: Immich creates one if none exists

File: docs/docs/features/libraries.md (hunk 2 above)
Comment (bo0tzz) [reply docs/docs/features/libraries.md]: I think the same is already noted at the start of this section, right?

> Note: Either a manual or scheduled library scan must have been performed to identify offline assets before this process will work.

Comment (aviv926) [reply docs/docs/features/libraries.md]: It seems so :)
I missed it

File: docs/docs/administration/oauth.md (hunk 1 above)
Comment (bo0tzz) [reply docs/docs/administration/oauth.md:11]: I'm still seeing this in the most recent version of the PR

**Review by bo0tzz — APPROVED** [review]

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**